### DISCUSSION TITLE
Release plans?

### DISCUSSION THREAD

**Discussion by bendb-instacart** [square/kotlinpoet#1047] [General]
Hi folks!

I'm curious about the plans for the next kotlinpoet release.  A few important (to me) bugfixes have landed since 1.7.2 shipped in August.  I'd like to incorporate them into my project, but would prefer not to add jitpack or snapshot dependencies.  Also, if there's any way I can help move things forward, I'd be grateful to know about it.

Thanks!

**Comment by Egorand** [comment]
We definitely have enough updates for 1.8.0, and it's been a while since the last release as you mentioned. I'll try to get to it on Monday, or by end of next week at the latest! Thanks for the ping & sorry for the long wait.

**Comment by Egorand** [comment]
Released! https://twitter.com/EgorAnd/status/1376661790474567680?s=20

**Reply by bendb-instacart** [reply]
Woo, awesome!  Thanks much.

**Comment by ZacSweers** [comment]
Locking this as resolved 👍
